### ISSUE TITLE
Multiple wiki support

### ISSUE THREAD

**Issue by Arkezis** [TFSExt/Forms#10]
Hi,
Thanks for your awesome module ! 
It really helps us :)

Do you know if you'll support multiple wiki ? 
It seems when I link to a wiki page, it only suggests me the "first" wiki in the list. 
I would like to be able to use any of my wikis.

Thanks :)

**Comment by awgeo** [comment]
I would also like to use this with multiple wikis, which would be helpful where some people are contributing to a Provisioned Wiki and others to Code Wiki within the same Project. This extension seems to work with whichever wiki was created first. Thank you.

**Comment by speedyco** [comment]
Yes, we have an Azure DevOps Repos Git/wiki-as-code published and would like to include that as well. Currently it shows the default internal one. Would that be an option?
